content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -240,42 +258,45 @@ For example:
 
 1. List all of the pods on a cluster in Protobuf format.
 
-        GET /api/v1/pods
-        Accept: application/vnd.kubernetes.protobuf
-        ---
-        200 OK
-        Content-Type: application/vnd.kubernetes.protobuf
-        ... binary encoded PodList object
+   ```console
+   GET /api/v1/pods
+   Accept: application/vnd.kubernetes.protobuf
+   ---
+   200 OK
+   Content-Type: application/vnd.kubernetes.protobuf
+   ... binary encoded PodList object
+   ```
 
 2. Create a pod by sending Protobuf encoded data to the server, but request a response in JSON.
 
-        POST /api/v1/namespaces/test/pods
-        Content-Type: application/vnd.kubernetes.protobuf
-        Accept: application/json
-        ... binary encoded Pod object
-        ---
-        200 OK
-        Content-Type: application/json
-        {
-          "kind": "Pod",
-          "apiVersion": "v1",
-          ...
-        }
+   ```console
+   POST /api/v1/namespaces/test/pods
+   Content-Type: application/vnd.kubernetes.protobuf
+   Accept: application/json
+   ... binary encoded Pod object
+   ---
+   200 OK
+   Content-Type: application/json
```

Comment:
mini nit: HTTP/1.1 usually has a blank line between headers and body; I'd include that here (it looks odd to me without it)